content/guides/cloud/smart-orchestration.md

```diff
@@ -42,41 +41,40 @@ time.**
 
 </Alert>
 
-### Toggle running of failed specs first
+### Activate Spec Prioritization in Cypress Cloud
 
 <Alert type="success">
 
 <strong class="alert-header"><Icon name="star"></Icon> Premium Cypress Cloud
 Feature</strong>
 
-**Running failed specs first** is a _Smart Orchestration_ feature available to
-users with a [Business Cypress Cloud plan](https://cypress.io/pricing).
+**Spec Prioritization** is a _Smart Orchestration_ feature available to users
+with a [Business Cypress Cloud plan](https://cypress.io/pricing).
```

Comment:
I think we can expand this callout to help the reader understand the impact ROAR can have on their decision about potentially upgrading their subscription, assuming they are a Team tier org. Something along the lines of, "Wondering if upgrading your Team tier subscription is worth it? You can always check out the Run Overview page to see how much time we predict these features would save you"

Reply:
I like this a lot, using the recommendations as another funnel.